.github/workflows/02-ci-stale.yml

```diff
@@ -0,0 +1,41 @@
+name: 🧹 Clean up stale issues and PRs
+
+on:
+  schedule:
+    - cron: "0 8 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    name: 🧹 Clean up stale issues and PRs
+    runs-on: ubuntu-latest
```

Comment:
#### Change Ubuntu-latest to Ubuntu-20.04

Using the latest version may generate an error in the future

```suggestion
    runs-on: ubuntu-20.04
```